enterprise/internal/own/background/background.go

```diff
@@ -28,19 +28,20 @@ import (
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
-type JobTypeID int
+// SignalJob should match the configuration name in the database table own_signal_configurations. This is used to match implementations to configuration.
```

Comment:
let's start with `SignalName`:
```suggestion
// SignalName should match the configuration name in the database table own_signal_configurations. This is used to match implementations to configuration.
```